game.js

```diff
@@ -67,7 +70,28 @@ const game = {
         ],
     },
     actions: {
-          hurtPlayer(amount, shouldSpeak = true) {
+        craftItem(itemName1, itemName2) { //TODO: Make a different game mode where you have to craft the item 
+            const item1 = items.find((item) => item.name.toLowerCase() === itemName1.toLowerCase());
+            const item2 = items.find((item) => item.name.toLowerCase() === itemName2.toLowerCase());
+            if (!(item1 && item2)) {
+              console.log('Missing or unknown item');
+              return;
+            }
+            const recipe = game.state.recipes.find((recipe) => recipe.ingredients.includes(item1.id) && recipe.ingredients.includes(item2.id));
+            if (!recipe) {
+              console.log(`${item1.name} can not be crafted with ${item2.name}`)
+              return;
+            }
+            if (!game.state.player.inventory.includes(item1.id && item2.id)) { console.log(`You do not have the items that you wish to craft with, make sure these items are in your inventory first...`); return; }
```

Comment:
You need to write two separate includes checks for this one. I’ll show you once you’re ready